arm-documentdb/2015-04-08/examples/DocumentDBDatabaseAccountListKeys.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "accountName": "ddb1",
+    "resourceGroupName": "rg1",
+    "api-version": "2015-04-08",
+    "subscriptionId": "subid"
+  },
+  "responses": {
+    "200": { 
+      "body": {
+        "primaryMasterKey": "primaryMasterKey",
+        "secondaryMasterKey": "secondaryMasterKey",
+        "primaryReadonlyMasterKey": "primaryReadonlyMasterKey",
+        "secondaryReadonlyMasterKey": "secondaryReadonlyMasterKey"
```

Comment:
General comment: please don't flatten properties. Flattening happens at the sdk level. On the wire things are the same as mentioned in the swagger spec.
So this should be:
```json5
"body": {
        "primaryMasterKey": "primaryMasterKey",
        "secondaryMasterKey": "secondaryMasterKey",
        "properties": {
          "primaryReadonlyMasterKey": "primaryReadonlyMasterKey",
          "secondaryReadonlyMasterKey": "secondaryReadonlyMasterKey"
        }
      }
```